packages/services/f-feature-management/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@justeat/f-feature-management",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
This is probably worth a minor version rather than a patch

Reply:
The package.json was supposed to be 0.2.1 previously as in the change log but I made a mistake. Maybe I’ll go to 0.4 now

Reply:
Done